python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -46,94 +46,66 @@ weights : sequence of float
 
 Notes
 -----
-Consider a physical model :math:`g` depending
-on a set random variables :math:`\vect{X}=(X_1, \dots, X_{n_X})^T`.
-We call functional chaos expansion the class of spectral methods which gathers
-all types of response surface that can be seen as a projection of the physical
-model in an orthonormal basis. This class of methods uses the Hilbertian space
-(square-integrable space: :math:`L^2`) to construct the response surface.
 
-The functional chaos representation implemented in OpenTURNS is the
-polynomial chaos (PC) expansion. It allows one to cast the response as a
-converging series featuring orthonormal polynomials.
+The polynomial chaos expansion enables to approximate the output random variable of interest :math:`\vect{Y} = g(\vect{X}) \in \Rset^p` where :math:`\vect{X}` is the input random vector of dimension *n* by the  response surface :
 
-Consider that the output random variable of interest
-:math:`\vect{Y} = g(\vect{X})` has a finite variance, i.e.
-:math:`\Var{g(\vect{X})} < \infty`
+.. math::
 
-Then it is shown that :math:`\vect{Y}` may be expanded onto the PC basis as
-follows:
+    \tilde{\vect{Y}} = \sum_{k \in K} \vect{\alpha}_k \Psi_k \circ T(\vect{X})
 
-.. math::
+where :math:`\vect{\alpha}_k \in \Rset^p`, *T* is an isoprobabilistic transformation which maps the multivariate distribution of :math:`\vect{X}` into the multivariate distribution :math:`\mu = \prod_{i=1}^n \mu_i`, and :math:`(\Psi_k)_{k \in \Nset}` a multivariate polynomial basis of :math:`\cL^2_{\mu}(\Rset^n,\Rset)` which is orthonornal according to the distribution :math:`\mu`. *K* is a finite subset of :math:`\Nset`. 
```

Comment:
That line (and others) is definetely too long, I have to drag the pointer on github to see it.